Assets/Scripts/EOSLobbyManager.cs

```diff
@@ -525,30 +525,59 @@ private bool IsLobbyNotificationValid(NotifyEventHandle handle)
         }
 
         // Helper method to keep the code cleaner
+
+        /// <summary>
+        /// Use to access functionality of [EOS_Lobby_AddNotifyLobbyUpdateReceived](https://dev.epicgames.com/docs/api-ref/functions/eos-lobby-add-notify-lobby-update-received)
+        /// The callback will only run if a listener is subscribed, which is done in <see cref="SubscribeToLobbyUpdates"/>.
+        /// </summary>
+        /// <param name="lobbyInterface">Handle to the lobby interface.</param>
+        /// <param name="notificationFn">Callback to receive notification when lobby member update is received</param>
         private ulong AddNotifyLobbyUpdateReceived(LobbyInterface lobbyInterface, OnLobbyUpdateReceivedCallback notificationFn)
         {
             var options = new AddNotifyLobbyUpdateReceivedOptions();
             return lobbyInterface.AddNotifyLobbyUpdateReceived(ref options, null, notificationFn);
         }
 
+        /// <summary>
+        /// Use to access functionality of [EOS_Lobby_AddNotifyLobbyMemberUpdateReceived](https://dev.epicgames.com/docs/api-ref/functions/eos-lobby-add-notify-lobby-member-update-received)
+        /// The callback will only run if a listener is subscribed, which is done in <see cref="SubscribeToLobbyUpdates"/>.
+        /// </summary>
+        /// <param name="lobbyInterface">Handle to the lobby interface.</param>
+        /// <param name="notificationFn">Callback to receive notification when lobby member update is received</param>
         private ulong AddNotifyLobbyMemberUpdateReceived(LobbyInterface lobbyInterface, OnLobbyMemberUpdateReceivedCallback notificationFn)
         {
             var options = new AddNotifyLobbyMemberUpdateReceivedOptions();
             return lobbyInterface.AddNotifyLobbyMemberUpdateReceived(ref options, null, notificationFn);
         }
 
+        /// <summary>
+        /// Use to access functionality of [EOS_Lobby_AddNotifyLobbyMemberStatusReceived](https://dev.epicgames.com/docs/api-ref/functions/eos-lobby-add-notify-lobby-member-status-received)
+        /// The callback will only run if a listener is subscribed, which is done in <see cref="SubscribeToLobbyUpdates"/>.
+        /// </summary>
+        /// <param name="lobbyInterface">Handle to the lobby interface.</param>
+        /// <param name="notificationFn">Callback to receive notification when lobby member update is received</param>
```

Comment:
Another copy-paste error - it should be "Callback to receive notification when lobby member status update is received." This block also excludes comments documenting the return value.

Reply:
Return value added.

Should it be "Callback to receive notification when lobby member status update is received" or "Callback to receive notification when lobby member status is received" (no update) given the name of the function?

Reply:
I would say whichever makes the most sense to you - I think there are decent arguments to be made for either - so long as it indicates that it happens when the lobby member status has changed.

Reply:
Feel free to mark this conversation as resolved.